src/Cli/dotnet/commands/dotnet-nuget/NuGetCommandParser.cs

```diff
@@ -73,5 +74,29 @@ private static Command GetPushCommand()
 
             return pushCommand;
         }
+
+        private static Command GetVerifyCommand()
+        {
+            var verifyCommand = new Command("verify");
+
+            verifyCommand.AddArgument(new Argument<IEnumerable<string>>() { Arity = ArgumentArity.OneOrMore });
+
+            verifyCommand.AddOption(new Option<bool>("--all"));
+            verifyCommand.AddOption(new ForwardedOption<IEnumerable<string>>("--certificate-fingerprint", "certificatefingerprint")
+                .ForwardAsMany(o => ForwardedArguments("--certificate-fingerprint", o))
+                .AllowSingleArgPerToken());
+            verifyCommand.AddOption(CommonOptions.VerbosityOption());
+
+            return verifyCommand;
+        }
+
+        public static IEnumerable<string> ForwardedArguments(string token, IEnumerable<string> arguments)
```

Comment:
I'm not sure I understand what this is needed for, are you trying to format the arguments such that each one is preceded by `--certificate-fingerprint`?

Reply:
Thanks for the review comment. This is where I need help from the `sdk` team.

`--certificate-fingerprint` option may be supplied multiple times to provide multiple fingerprints.
For example: `dotnet nuget verify foo.nupkg --certificate-fingerprint CE40881FF5F0AD3E58965DA20A9F571EF1651A56933748E1BF1C99E537C4E039 --certificate-fingerprint EC10992GG5F0AD3E58965DA20A9F571EF1651A56933748E1BF1C99E537C4E027`

I copied logic from `dotnet list package` command which allows users to specify multiple frameworks by repeating the option multiple times. `For example: --framework netcoreapp2.2 --framework netstandard2.0`

https://github.com/dotnet/sdk/blob/234a65c6398d8d3335553bdea9d06e0e47c89d69/src/Cli/dotnet/commands/dotnet-list/dotnet-list-package/ListPackageReferencesCommandParser.cs#L22-L26

Currently `ForwardedArguments` method code is duplicated in `ListPackageReferencesCommandParser.cs -> existing` and `NuGetCommandParser.cs -> added in this PR`. Best option is to takeout the method from these classes and add it as a utility so that it can be reused. I couldn't figure out the best place in the source code. I appreciate your feedback.

Reply:
Agreed, reusing this would be great. It could go in https://github.com/dotnet/sdk/blob/main/src/Cli/dotnet/OptionForwardingExtensions.cs

Reply:
Added `ForwardedArguments` as an extension method and changed the callers to call the new API. Added strings to the resources file looking at other commands' implementation.